Haven't figured out why the CI scripts don't run on PRs (probably missing a trigger), but the branch has a few failed tests.

<details>
<summary>Test Output</summary>

```
> hubot-github-deployments@1.2.14 test
> mocha --require coffee-script/register "test/**/*.coffee" --reporter spec



  hubot-github-deployments
    ✔ responds with the list of recent deployments (1003ms)
    ✔ responds with the list of recent deployments for a specific repo (1000ms)
    ✔ gets the status of a single deployment (1001ms)
    ✔ gets the status of a single deployment for a specific repo (1001ms)
[Thu Sep 28 2023 20:23:53 GMT-0500 (Central Daylight Time)] ERROR ReferenceError: match is not defined
    at TextListener.callback (/Users/stephen/Sites/hubot-github-deployments/src/github-deployments.coffee:117:37)
    at executeListener (/Users/stephen/Sites/hubot-github-deployments/node_modules/hubot/src/listener.js:79:16)
    at allDone (/Users/stephen/Sites/hubot-github-deployments/node_modules/hubot/src/middleware.js:56:7)
    at /Users/stephen/Sites/hubot-github-deployments/node_modules/async/lib/async.js:254:13
    at Object.async.eachSeries (/Users/stephen/Sites/hubot-github-deployments/node_modules/async/lib/async.js:130:20)
    at async.reduce (/Users/stephen/Sites/hubot-github-deployments/node_modules/async/lib/async.js:248:15)
    at processTicksAndRejections (node:internal/process/task_queues:78:11)

    1) creates a deployment
[Thu Sep 28 2023 20:23:54 GMT-0500 (Central Daylight Time)] ERROR ReferenceError: match is not defined
    at TextListener.callback (/Users/stephen/Sites/hubot-github-deployments/src/github-deployments.coffee:117:37)
    at executeListener (/Users/stephen/Sites/hubot-github-deployments/node_modules/hubot/src/listener.js:79:16)
    at allDone (/Users/stephen/Sites/hubot-github-deployments/node_modules/hubot/src/middleware.js:56:7)
    at /Users/stephen/Sites/hubot-github-deployments/node_modules/async/lib/async.js:254:13
    at Object.async.eachSeries (/Users/stephen/Sites/hubot-github-deployments/node_modules/async/lib/async.js:130:20)
    at async.reduce (/Users/stephen/Sites/hubot-github-deployments/node_modules/async/lib/async.js:248:15)
    at processTicksAndRejections (node:internal/process/task_queues:78:11)

    2) creates a deployment for a specific repo
    ✔ lists the target environments (1001ms)
    ✔ lists all branches (1001ms)
    ✔ lists all branches for a specific repo (1001ms)
    ✔ lists branches matching search (1001ms)
    ✔ lists branches matching search for a specific repo (1002ms)
    ✔ lists branches matching search, no results (1002ms)
    ✔ lists branches matching search for a specific repo, no results (999ms)


  11 passing (13s)
  2 failing

  1) hubot-github-deployments
       creates a deployment:

      AssertionError: expected [ [ 'alice', …(1) ] ] to deeply equal [ Array(2) ]
      + expected - actual

         [
           "alice"
           "@hubot deploy master to production"
         ]
      +  [
      +    "hubot"
      +    "stephenyeargin deployed master to production"
      +  ]
       ]
      
      at Timeout._onTimeout (test/github-deployments_test.coffee:114:38)
      at listOnTimeout (node:internal/timers:559:17)
      at processTimers (node:internal/timers:502:7)
  

  2) hubot-github-deployments
       creates a deployment for a specific repo:

      AssertionError: expected [ [ 'alice', …(1) ] ] to deeply equal [ Array(2) ]
      + expected - actual

         [
           "alice"
           "@hubot deploy master to production for someone/somewhere"
         ]
      +  [
      +    "hubot"
      +    "stephenyeargin deployed master to production"
      +  ]
       ]
      
      at Timeout._onTimeout (test/github-deployments_test.coffee:133:38)
      at listOnTimeout (node:internal/timers:559:17)
      at processTimers (node:internal/timers:502:7)
  
```

</details>

--- a/.github/workflows/nodejs.yml
+++ b/.github/workflows/nodejs.yml
@@ -1,6 +1,9 @@
 name: Node CI
 
-on: [push]
+on: 
+  push:
+    branches:
+      - "**"
 
 jobs:
   build:

--- a/.github/workflows/nodejs.yml
+++ b/.github/workflows/nodejs.yml
@@ -1,9 +1,8 @@
 name: Node CI
 
-on: 
+on:
   push:
-    branches:
-      - "**"
+  pull_request:
 
 jobs:
   build:

--- a/src/github-deployments.coffee
+++ b/src/github-deployments.coffee
@@ -114,7 +114,7 @@ module.exports = (robot) ->
     unless checkConfiguration(res)
       return;
 
-    app = getRepoInfo(res.match[3], match[4])
+    app = getRepoInfo(res.match[3], res.match[4])
     if !app?
       res.send "Missing configuration: HUBOT_GITHUB_REPO"
       return false

